source/images/buyers-guide/filter/checkbox-selected.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's `svgo` this ✨

Reply:
agreed, but fun fact: the npm scripts for svgo etc are weirdly "you can't run these on windows".

Reply:
updated the svgo task, because it's had the `--recursive` and `--folder` options for a while now so we can do this without find/xarg \o/